utils/utils.go

```diff
@@ -259,10 +295,42 @@ func WatchCerts() {
 	}
 }
 
+// parseSSHOptions parses options from ssh.ParseAuthorizedKey format to our map format for SSHAuthKey.
+func parseSSHOptions(options []string) map[string][]string {
+	ret := make(map[string][]string)
+	for _, o := range options {
+		values := make([]string, 0)
+		optionSplit := strings.Split(o, "=")
+
+		if len(optionSplit) == 1 {
+			ret[optionSplit[0]] = values
+		}
+		if len(optionSplit) == 2 {
+			v, ok := ret[optionSplit[0]]
+			if ok {
+				values = v
+			}
+
+			optionVal := optionSplit[1]
```

Comment:
Try to `return` fast in order to have less nesting.

Reply:
Fast `return` does not make sense in this cycle. Maybe `break` would do, but in this case, it would make readability worse instead of better, don't you think?

Reply:
bump